configs/opencv-ovms/gst_capi/pipelines/face_detection/main.cpp

```diff
@@ -943,15 +926,19 @@ int main(int argc, char** argv) {
     }
 
     if (!stringIsInteger(argv[2]) || !stringIsInteger(argv[3]) || !stringIsInteger(argv[4])
-        || !stringIsInteger(argv[5])) {
+        || !stringIsInteger(argv[5]) || !stringIsInteger(argv[6]) || !stringIsInteger(argv[7])) {
```

Comment:
please also add the length check. what if you don't have 5 or 6 or 7, then the app will crash